app/lib/actions/pulp/abstract_async_task.rb

```diff
@@ -79,7 +79,7 @@ def external_task
         output[:pulp_tasks]
       end
 
-      def cancel!
+      def cancel!(raise_error = true)
```

Comment:
I contemplated passing the error in and that seemed even worse (considering this method isn't the one throwing the exception), and i really don't like catching and rethrowing exceptions, so i will move to a 2nd function the actual cancel logic.